eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -2,7 +2,7 @@
 . "${PSScriptRoot}\logging.ps1"
 . "${PSScriptRoot}\SemVer.ps1"
 
-$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?)(\s+(?<releaseStatus>\(Unreleased\)|\(\d{4}-\d{2}-\d{2}\)))?)"
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?)(\s+(?<releaseStatus>\(.*\))?)"
```

Comment:
Why loosen this? If we don't match the title then can we just leave the entry alone?

Reply:
We will be using it in other places though. e.g. in collect changelog to pull out released for the required date. I guess we have to enforce it at some point to make sure its valid date format. We also loosened it for changelog verification before release.

Reply:
We were depending on the regex to enforce our guidelines but if you loosen it here then we should add extra validation in other places in the helper scripts to ensure people still follow the guidelines. 

However I'm not sure why we cannot simply ignore any sections that aren't in the valid format.

Reply:
I don't know of any other place we are using this regex. Looks like its only used to grab the release status/date. We do enforce that its a valid date before releasing https://github.com/Azure/azure-sdk-tools/blob/e6d24dce8fe81b327ba806cf6f8ef2d1935f3eca/eng/common/scripts/ChangeLog-Operations.ps1#L123 so I think we should be fine with loosening it here. We'll be able to accommodate older entries that are not correct while making sure new entries are.

Reply:
Ok lets see if this helps with the older entries. How do we plan to handle the version numbers with preview but no number?